app/lib/mini-course.cjsx

```diff
@@ -17,81 +16,73 @@ module.exports = React.createClass
 
   statics:
     find: ({workflow}) ->
-      # Prefer fetching the tutorial for the workflow, if a workflow is given.
+      # Prefer fetching the tutorial for the workflow, so we know which one to fetch if multiple exist.
       if workflow?
         apiClient.type('tutorials').get workflow_id: workflow.id, kind: "mini-course"
           .then ([minicourse]) ->
             minicourse
```

Comment:
Ahhh, I get it. Can we make the mini-course selection control out of radio buttons then?

Looks like it's the same for tutorials.